static/css/styles.css

```diff
@@ -259,25 +285,25 @@ p a:hover {
   display: flex;
   align-items: center;
   gap: 12px;
-  opacity: 0;
-  pointer-events: none;
-  transition: opacity 0.2s ease;
-}
-
-html.js-loaded .header-controls {
   opacity: 1;
   pointer-events: auto;
+  transition: opacity 0.2s ease;
 }
 
 /* Toggle Switch Styles */
 .toggle-switch {
+  visibility: hidden;
```

Comment:
The toggle switch is hidden by default and must be made visible in JS. This ensures everything works and looks right when JS is turned off.